src/read_concern.ts

```diff
@@ -7,6 +7,12 @@ export enum ReadConcernLevel {
   snapshot = 'snapshot'
 }
 
+/** @public */
+export type ReadConcernLevelLike = ReadConcernLevel | keyof typeof ReadConcernLevel;
+
+/** @public */
+export type ReadConcernLike = ReadConcern | { level: ReadConcernLevelLike } | ReadConcernLevelLike;
```

Comment:
We support "plain objects" for `readPreference` and `writeConcern`, in those cases it makes sense because they have more than one key. This is just to maintain consistency between those options and a possible interoperation when reading the old docs.

https://mongodb.github.io/node-mongodb-native/3.6/api/MongoClient.html

Reply:
I just moved `{ level: ReadConcernLevel }` from the `fromOptions`, this was already here.

Reply:
hm, yeah I think we should stop supporting those, but that sounds like another ticket. Can you make ticket for removing support for plain objects for those properties, and add it to the `MongoClientOptions` epic?

Reply:
Created => NODE-2836